typescript/src/package.json

```diff
@@ -28,7 +28,8 @@
     "md5": "^2.3.0",
     "uncrypto": "^0.1.3",
     "viem": "^2.21.26",
-    "zod": "^3.24.4"
+    "zod": "^3.24.4",
+    "bigint-buffer": "npm:bigint-buffer-fixed@^1.1.6"
```

Comment:
nit: can we alphabetize (pnpm will do it automatically if you run `pnpm add bigint-buffer@npm:bigint-buffer-fixed@^1.1.6` from `typescript/src`